### DISCUSSION TITLE
Change machine clock will effect delay time

### DISCUSSION THREAD

**Discussion by leo-987** [rabbitmq/rabbitmq-delayed-message-exchange#168] [General]
I changed the server machine clock, and then produce some delay message, this is my test result:
```
15/06/2021 15:34:56, 15/06/2021 14:35:29, <BasicProperties(['content_type=application/json', 'delivery_mode=2', "headers={'x-delay': 3600000}", 'message_id=python'])>, time_diff=3567
```
14:35:29 send, 15:34:56 receive, total delay time is 3567 seconds. Is this normal?

**Comment by michaelklishin** [comment]
Most time-based systems assume that the clock does not drift. So does this plugin. It is entirely unaware of any system clock changes and won't be able to meaningfully react to them.

**Reply by leo-987** [reply]
Got it, thanks.

**Comment by OleksiiD** [comment]
hi @michaelklishin 

What about the scenario when the system switches to/from Daylight Saving Time?

Will it have any negative impact on plugin functionality (unexpected behavior etc.)?
